test/core/end2end/tests/shutdown_finishes_tags.c

```diff
@@ -78,7 +78,7 @@ static void test_early_server_shutdown_finishes_tags(
   grpc_end2end_test_fixture f = begin_test(
       config, "test_early_server_shutdown_finishes_tags", NULL, NULL);
   cq_verifier *cqv = cq_verifier_create(f.cq);
-  grpc_call *s = (void *)1;
+  grpc_call *s = (grpc_call *)1;
```

Comment:
This in general should be a double-cast: `(grpc_call *)(uintptr_t)1`

Reply:
Done! Thanks!